test_communication/test/test_messages_c.cpp

```diff
@@ -174,7 +174,7 @@ class CLASSNAME (TestMessagesFixture, RMW_IMPLEMENTATION) : public ::testing::Te
           fini_message(&message);
         });
 
-        rcl_wait_set_t wait_set = rcl_get_zero_initialized_wait_set();
+        wait_set = rcl_get_zero_initialized_wait_set();
```

Comment:
Would it be better/possible to narrow the scope above to avoid the shadowing (I assume that was the issue here)?